scripts/motor-expert-screen

```diff
@@ -96,17 +96,20 @@ elif [ "${rtyp}" == 'motor' ]; then
     caget "$PREFIX:PN" > /dev/null 2>&1
     if [ $? -eq 0 ]; then
         /reg/g/pcds/package/epics/3.14//modules/pcds_motion/${OLD_VERSION}/launch-motor.sh "$PREFIX" > /dev/null 2>&1 &
-    elif [[ $PREFIX == *"MCS2"* ]]; then # smaracts have "motor" rtyp, so we catch them with the PREFIX
-        enc=$(caget $PREFIX.UEIP)
-        if [[ $enc == *"Yes"* ]]; then
-            edm -x -eolc -m "MOTOR=$PREFIX" mcs2_main.edl > /dev/null 2>&1 &
-        else
-            edm -x -eolc -m "MOTOR=$PREFIX" mcs2_openloop.edl > /dev/null 2>&1 &
-        fi
     else
-        cd /reg/neh/home/klg/epics/ioc/common/aerotech/current/motorScreens
-        #cd /reg/neh/home4/mcbrowne/trunk2/ioc/common/aerotech/current/motorScreens
-        edm -x -eolc -m "MOTOR=${PREFIX}" ens_main.edl >& /dev/null &
+	caget "$PREFIX:LSCO_RBV" > /dev/null 2>&1
```

Comment:
Can you give an example of an IOC where there is a LSCO_RBV PV but not MCS2 in the PV name? Here's the result of my search:
```
zlentz@psbuild-rhel7-01:~$ grep_pv LSCO_RBV
/reg/d/iocData/ioc-lm1k2-mcs2-03/iocInfo
LM2K2:MCS2:03:m1:LSCO_RBV, "int64in"
LM2K2:MCS2:03:m2:LSCO_RBV, "int64in"

zlentz@psbuild-rhel7-01:~$
```
maybe grep_pv doesn't show aliases?

Reply:
@KaushikMalapati I think a better PV to use for this would be `PTYPE` or `PTYPE_RBV`. The `LSCO` PVs are recently added, and so IOCS based on older releases will not have this. The `PTYPE` PVs are specific to SmarAct, and have always been a part of the IOC.

Reply:
I didn't know that PTYPE was specific to smaract, I've changed it to use PTYPE_RBV instead of LSCO_RBV